teamB/seongmin/westagram/app.js

```diff
@@ -58,7 +58,17 @@ app.post('/users/sign-up', async(req, res) => {
     );
 
     res.status(201).json({ mesaage: "userCreated"});
-})
+});
+
+app.post('/addPost', async(req, res) => {
```

Comment:
이미 http method 자체가 데이터를 추가하는 의미를 갖고 있기 때문에, target 부분에는 동사를 제외하고 명사만 사용해주세요.

```suggestion
app.post('/post', async(req, res) => {
```